src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -566,11 +569,30 @@ def _library_dep_order(library: str):
             case _:
                 return 3
 
-    def _upload_wheel(self):
+    def _upload_wheelhouse(self, path: Path) -> list[str]:
+        """Upload a wheelhouse.
+
+        TODO
+        ----
+        Move this method into the WheelsV2 class.
+        """
+        # We can not upload a .zip file to the workspace as the platform fails when extracting all files inside the zip
+        # archive, therefore, we upload the wheels separately.
+        remote_paths = []
+        with tempfile.TemporaryDirectory() as directory, zipfile.ZipFile(path, "r") as zip_ref:
```

Comment:
what about `requirements.txt`?

Reply:
Not required in the workflow, but we can add it